### PR TITLE
Improvements to DE translation

### PR DESCRIPTION
**Pull request by ketzu**
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes

Inconsistency and an error in the german translation.
Gitter -> Raster (it was called Raster in every other section and is, as far as I can tell, called raster in other applications)

Festlegen - bis -> Setzen - auf
As "festlegen bis" makes no sense at all in the context of a change node.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by knolleary** [comment]
@hobbyquaker @jwende please review these proposed changes to the .de translations.

**Comment by jwende** [comment]
@ketzu Glad that you are going to contribute - but can we discuss this on slack (#2193 i18n-german) or our translation issues github (https://github.com/jwende/nodered-german)  before (so we could switch to german ;-)) 
1) Gitter->Raster - I see this too 
2) not clear (need a pointer to find this in the UI)

Cheers
J.

**Comment by knolleary** [comment]
Can I check where this has got to? Have there been further discussions on the translations and are there any updates needed?

**Comment by dceejay** [comment]
@hobbyquaker @jwende ding ding - heads up guys. We are trying to finalise the 1.0 release of Node-RED - so would really appreciate some input on these translations - or they will miss the boat and be in a fix later on.
Thanks

**Comment by floookay** [comment]
@jwende @knolleary You can find the second fix in the properties menu of the change node:
image
I am in favor of this change since this confused me as well until I looked at the English translation. "Setzen ... auf ..." would clear it up.

File: packages/node_modules/@node-red/nodes/locales/de/messages.json [hunk 1]
```diff
@@ -596,11 +596,11 @@
       "regex" : "Reguläre Ausdrücke verwenden"
     },
     "action" : {
-      "set" : "Festlegen",
+      "set" : "Setze",
```
Comment (floookay) [inline packages/node_modules/@node-red/nodes/locales/de/messages.json]: Maybe change this line to "Setzen" to make it consistent with the other three options of the change node.

Comment (patricksadowski) [reply packages/node_modules/@node-red/nodes/locales/de/messages.json:599]: @ketzu Are you still going to make this change?

**Pull request merged**